test/conformance/testdata/conformance.txt

```diff
@@ -36,6 +36,7 @@ test/e2e/apps/statefulset.go: "should perform canary updates and phased rolling
 test/e2e/apps/statefulset.go: "Scaling should happen in predictable order and halt if any stateful pod is unhealthy"
 test/e2e/apps/statefulset.go: "Burst scaling should run to completion even with unhealthy pods"
 test/e2e/apps/statefulset.go: "Should recreate evicted statefulset"
+test/e2e/apps/replica_set.go: "should surface a failure condition on a common issue like exceeded quota"
```

Comment:
this is listed on the wrong line

Reply:
see https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/80148/pull-kubernetes-bazel-test/1154125960350535681#0:build-log.txt%3A56